### ISSUE TITLE
A broken link in addons/actions/ADVANCED.md 

### ISSUE THREAD

**Issue by yhor1e** [storybookjs/storybook#12753]
**Describe the bug**
A broken link.

https://github.com/storybookjs/storybook/blame/230d1f18be0e688e868dfc65be14f4deb7a2b7b8/addons/actions/ADVANCED.md#L11

**To Reproduce**

1. Go to https://github.com/storybookjs/storybook/blob/next/addons/actions/ADVANCED.md
2. Click on 'issues#29' link
3. See 404 page

**Expected behavior**

Display the page correctly.

**Comment by jonniebigodes** [comment]
@yhor1e thank you for bringing this to our attention. It seems that with a recent change in the storybook organization the addons were consolidated into one monorepo and with that it seems that the repo for that particular addon is no longer around. I digged around and could not find it anymore. That's why the 404 in the documentation. I've made a pull request that addresses your issue and the 404 is no longer displayed.

Stay safe
